### PR TITLE
New package: ydotool-1.0.1

### PR DESCRIPTION
**Pull request by JohnGebbie**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures:
  - aarch64-musl
  - armv7l
  - armv6l-musl

I started with https://github.com/void-linux/void-packages/pull/31237

Closes https://github.com/void-linux/void-packages/issues/31163

### CODE REVIEW

**Comment by JohnGebbie** [comment]
I pushed to make my branch up to date.

**Comment by RunningDroid** [comment]
It looks like the manpage wasn't updated after the refactor: https://github.com/ReimuNotMoe/ydotool/issues/147

[help output for the key command](https://github.com/ReimuNotMoe/ydotool/blob/master/Client/tool_key.c#L39-L59)
[relevant section of the manpage](https://github.com/ReimuNotMoe/ydotool/blob/master/manpage/ydotool.1.scd#keyboard-commands)

File: srcpkgs/ydotool/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'ydotool'
+pkgname=ydotool
+version=1.0.1
+revision=1
+build_style=cmake
+hostmakedepends="scdoc"
```
[the remark this answers is not in the document]
Comment (JohnGebbie) [reply srcpkgs/ydotool/template]: I've added it, thanks.

**Comment by JohnGebbie** [comment]
Sorry I don't know how it closed, I meant to update the branch and somehow clobbered everything.

**Comment by JohnGebbie** [comment]
We've got a yodotoold service that runs as the _yodotoold user so the yodotool command works without root, but it'd be good to the able to run the yodotoold command without root as well (provided you're in the "input" group).
I'm not sure how to do that.

**Comment by JohnGebbie** [comment]
Actually not quite, the ydotool manpage should say ydotoold needs to be running, as it does since the refractor.

**Comment by JohnGebbie** [comment]
Ok that's it for realsies with up to date manpages.

**Comment by JohnGebbie** [comment]
Found an issue, mousemouse --absolute is wrong due to mouse acceleration: https://github.com/ReimuNotMoe/ydotool/issues/158

**Comment by JohnGebbie** [comment]
That's a fix for the mouse acceleration and I've updated the branch.

**Comment by JohnGebbie** [comment]
That's it ready.

**Pull request closed without merging**